spec/02-integration/20-wasm/06-clustering_spec.lua

```diff
@@ -177,6 +177,8 @@ describe("#wasm - hybrid mode #postgres", function()
             res = res:read_body(),
           }
         end)
+        .with_timeout(60)
```

Comment:
re: all of these timeout changes

This is to account for the increased NGINX startup time incurred from loading the new `datakit` module. Locally I'm seeing about ~3.6s pretty consistently, but timing in CI (where we are often saturating the CPU) is slower and very inconsistent, so a large timeout is selected to keep the test from being flaky.

There is some work in progress to mitigate the slow startup time: https://github.com/Kong/ngx_wasm_module/pull/536

Reply:
update: because module loading is an init_worker thing, the privileged agent worker is _also_ delayed on startup, so this affects how long it takes for the DP to establish a connection to the CP. I believe that is why this test suite is struggling so much more than others. Changes:

1. added some more logfile assertions to watch for a module load event (technically this only guarantees that 1/N processes has finished init_worker, but it's still somewhat helpful).
2. adjusted the worker procs to 2
3. disabled bundled filters in this test suite because `datakit` is the biggest source of slow init_worker